### DISCUSSION TITLE
Translations of tutorials within the learnr package: where, how

### DISCUSSION THREAD

**Discussion by xavidp** [rstudio/learnr#784] [Q&A] [answered]
First of all, congrats for the learnr package and all the content and tools you created already! (a-w-e-s-o-m-e!)

Then my question. 

I'm considering to translate some of the tutorials of the learnr package to a local language (maybe Spanish), so that I can use them in a course I'm preparing for local civil servants in my organization that we would need them introduced into using R (and tweaking some tools were are developing based on R/Rmd scripts).

But I wonder what's the best way to proceed, so that the effort can be used/extended by others in the future, etc. Shall I fork the learnr repo, and just translate the tutorials*.Rmd files I'm interested in? (the 5 first ones starting with "ex-", probably, as a starting point).
 
The fact is that they are linked to specific chapters of the book r4ds - v1, and that book has been translated to Spanish also ( https://es.r4ds.hadley.nz/ ), so that the translated tutorials wouldn't be exact translations of the English version but adapted to point and reuse the examples in the Spanish version of the r4ds (v1) book.

Therefore, what's the best way to proceed?

Thanks in advanced

**Comment by gadenbuie** [comment, marked as answer]
Hi @xavidp! Thanks for your post, I'm really glad to hear learnr is helpful to you and helping you teach others!

I can think of a few things we could do to make full-tutorial translations work well with learnr, but the first thing I'd need is to have actual full translations 😄 If you wanted to contribute those, we'd be very happy to include them. 

I think the way to get started there would be to create a copy of the folder containing the tutorial you're translating, adding the language code to the end of the parent folder's name. If you do this work in a branch on a fork of `rstudio/learnr`, you could create a PR and we can collaborate in that PR to get the final details just right.

Your folder structure would look something like this:

```
inst
└── tutorials
    ├── ex-data-basics
    │   ├── ex-data-basics.Rmd
    │   └── images
    │       └── flights.png
    └── ex-data-basics_es
        ├── ex-data-basics.Rmd
        └── images
            └── flights.png
```

**Reply by xavidp** [reply]
perfect, I'll do so. Stay tuned (expect some PR to happen within the next hours - the 5 tutorials I was interested in have been translated already within the last week)

**Reply by xavidp** [reply]
done the PR in https://github.com/rstudio/learnr/pull/788
